pombola/core/views.py

```diff
@@ -325,7 +325,7 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
     # Order by place name unless ordering by person name is requested:
     order = 'place'
     if request.GET.get('order') == 'name':
-        order = 'person__legal_name'
+        order = 'person__sort_name'
```

Comment:
We make sure that when a model is saved, the sort name is set to a (somewhat) sensible default here: https://github.com/mysociety/pombola/blob/master/pombola/core/models.py#L506-L512 and when the field was introduced the migration set sort_name for everyone: https://github.com/mysociety/pombola/blob/7ca886ee4e44e2fafdaedfa929c9e81a69aedeb8/pombola/core/migrations/0055_auto__add_field_person_biography__add_field_person_national_identity__.py#L98-L103 so it should always have a value